### PR TITLE
Added "RemoveLayeredClothings" command

### PR DESCRIPTION
**Pull request by karl-police**
You currently can't remove layered accessories. "GetAccessories(false)" is used to retrieve the layered clothings from there.

@Expertcoderz

### CODE REVIEW

File: MainModule/Server/Commands/Moderators.lua [hunk 1]
```diff
@@ -892,6 +892,34 @@ return function(Vargs, env)
 				end
 			end
 		};
+		
+		RemoveLayeredClothings = {
+			Prefix = Settings.Prefix;
+			Commands = {"removelayeredclothings"};
```
Comment (watameln) [inline MainModule/Server/Commands/Moderators.lua:898]: pls make short subcommand :)

Comment (karl-police) [reply MainModule/Server/Commands/Moderators.lua:898]: huh?

Comment (ccuser44) [reply MainModule/Server/Commands/Moderators.lua:898]: It means that a command with a shorter name should exist because typing the full name is cumbersome

**Review by ccuser44 — COMMENTED** [review]
Or maybe instead all of the commands `RemoveLayeredClothings`, `RemoveShirt`, `RemovePants`, `RemoveTShirt` should be merged in to one single command?

**Comment by karl-police** [comment]
> Or maybe instead all of the commands `RemoveLayeredClothings`, `RemoveShirt`, `RemovePants`, `RemoveTShirt` should be merged in to one single command?

Not at all. Merging the commands would mean having to create a second argument to specify what to remove.

Not everyone wants to remove all of it at once.

Merging RemoveShirt, RemovePants and RemoveTShirt would also ruin people's command copy pastes.

**Pull request merged**